src/test/tests/pypiUtils.test.ts

```diff
@@ -48,6 +48,46 @@ describe('Pypi Utils Tests', async () => {
         createVirtualEnvironment();
     });
 
+    it('Match setup.py dependencies with regex', () => {
+        let map:Map<string, string | undefined> = PypiUtils.getSetupPyDirectDependencies(path.join(tmpDir.fsPath, 'regex', 'setupNoDepFound.py'))
```

Comment:
"map" is not an informative name. Map of what?
Same question for all new tests in this file.